deploy/kubeconfiggenerator.py

```diff
@@ -493,6 +493,8 @@ def flatten(yaml_contents, flattened, types_dict, prefix=''):
             inner_prop_dict = {}
             prop_dict = {'properties': inner_prop_dict}
             prop_dict['type'] = 'object'
+            #Allow 
+            prop_dict['additionalProperties'] = True
```

Comment:
Is this change for the requirement that you had identified regarding supporting empty annotations? 

If so, it will be better to separate it in a separate PR. That way it will be easier to merge it while we are still going through the build related changes.